### PR TITLE
CNV-37517: Add routes to diagnostic tab

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Adding routes to the diagnostic tab

## 🎥 Demo

image

image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/diagnostic/utils/utils.ts [hunk 1]
```diff
@@ -93,3 +93,16 @@ export const buildDataVolumeResources = (vm: V1VirtualMachine) =>
       },
     ]) || [],
   );
+
+export const createURLDiagnostic = (str: string, append: string): string => {
+  const urlSpitted = str.split('/');
+  if (
+    urlSpitted[urlSpitted.length - 1] === 'logs' ||
+    urlSpitted[urlSpitted.length - 1] === 'tables'
```
Comment (avivtur) [inline src/views/virtualmachines/details/tabs/diagnostic/utils/utils.ts:101]: can you use the consts here too? VirtualMachineDetailsTab.Tables and VirtualMachineDetailsTab.Logs

**Review by avivtur — COMMENTED** [review]
looks good just one small comment :)

File: src/views/virtualmachines/details/tabs/diagnostic/utils/utils.ts (hunk 1 above)
Comment (metalice) [reply src/views/virtualmachines/details/tabs/diagnostic/utils/utils.ts:101]: sure, done

**Review by avivtur — APPROVED** [review]

**Pull request merged**